bullet_train/lib/bullet_train/engine.rb

```diff
@@ -19,6 +19,12 @@
 rescue LoadError
 end
 
+begin
+  # And the same holds true for invisible_captcha
+  require "invisible_captcha"
```

Comment:
@julianrubisch why do we need to hoist the loading of the gem? It doesn't look like we've got `view_paths` that would conflict like in the case of `devise` and `showcase-rails`.

Reply:
If I only remembered! 😅

This PR has been dangling for a while, but I remember there _was_ a reason for it.

Can look into it when I'm back at a computer later this month.

Reply:
I just checked CI and it looks it's that `invisible_captcha` otherwise isn't available in the controller yet, when you do `invisible_captcha only: :create`. 

https://app.circleci.com/pipelines/github/bullet-train-co/bullet_train-core/841/workflows/b7e81028-49cf-4fab-9ef9-37148c15ddec/jobs/2589

So it's because we're inserting `invisible_captcha` when `Registrations::ControllerBase` is loaded, which means Bullet Train is loaded before `invisible_captcha`! Ok, that makes sense to me.

Reply:
Ah yes. Otherwise I probably wouldn't have wasted a commit on it 😅